app/components/steps_component/steps_component.scss

```diff
@@ -0,0 +1,108 @@
+@import 'base_component';
+
+.steps-component {
+  border-top: 3px solid govuk-colour('blue');
+  margin-bottom: govuk-spacing(6);
+  margin-top: govuk-spacing(3);
+
+  h2 {
+    margin-bottom: govuk-spacing(2);
+    margin-top: govuk-spacing(4);
+  }
+
+  &__steps {
+    counter-reset: steps-counter;
+    list-style: none;
+    margin-block-end: 0;
+    margin-block-start: 0;
+    padding-inline-start: 0;
+  }
+
+  &__step {
+    counter-increment: steps-counter;
+    padding-bottom: govuk-spacing(1);
+    padding-top: govuk-spacing(3);
+    position: relative;
+
+    h3 {
+      margin-bottom: 0;
+
+      &::before {
+        border: 2px solid $govuk-border-colour;
+        border-radius: 50%;
+        content: counter(steps-counter);
+        display: inline-block;
+        height: govuk-spacing(5);
+        line-height: govuk-spacing(5);
+        margin-right: govuk-spacing(4);
+        text-align: center;
+        width: govuk-spacing(5);
+      }
+
+      @include govuk-media-query($from: mobile) {
```

Comment:
shouldnt have a mediaquery in a media query like this. should have mobile styles and then overide them as viewport gets bigger